lib/process-services-cloud/src/lib/group/components/group-cloud.component.ts

```diff
@@ -81,26 +80,26 @@ export class GroupCloudComponent implements OnInit, OnChanges {
 
     /** Emitted when a group is selected. */
     @Output()
-    selectGroup: EventEmitter<GroupModel> = new EventEmitter<GroupModel>();
+    selectGroup: EventEmitter<IdentityGroupModel> = new EventEmitter<IdentityGroupModel>();
```

Comment:
With current TypeScript, you can simplify the assignment
```suggestion
    selectGroup = new EventEmitter<IdentityGroupModel>();
```